src/components/Boards/lib/generateBoardName.test.ts

```diff
@@ -0,0 +1,16 @@
+import generateBoardName from './generateBoardName';
+const MockDate = require('mockdate');
```

Comment:
`mockdate` might not be that valuable since we can do it ourselves, but I think it's nice for convenience

Reply:
That's why I added it 😄